rotkehlchen/api/v1/schemas.py

```diff
@@ -3127,3 +3128,75 @@ class ClearAvatarsCacheSchema(Schema):
         fields.String(required=True, validate=lambda x: x.endswith('.eth')),
         load_default=None,
     )
+
+
+class EthStakingHistoryStats(AsyncQueryArgumentSchema, OnlyCacheQuerySchema):
+    """Schema for quering history events"""
+    from_timestamp = TimestampField(load_default=Timestamp(0))
+    to_timestamp = TimestampField(load_default=ts_now)
+    location_labels = DelimitedOrNormalList(fields.String(), load_default=None)
+    validator_indices = DelimitedOrNormalList(fields.Integer(), load_default=None)
+
+    def __init__(self, db: 'DBHandler') -> None:
+        super().__init__()
+        self.db = db
+
+    @post_load
+    def make_history_event_filter(
+            self,
+            data: dict[str, Any],
+            **_kwargs: Any,
+    ) -> dict[str, Any]:
+        base_arguments = {
+            'from_ts': data['from_timestamp'],
+            'to_ts': data['to_timestamp'],
+        }
+        validator_indices = data['validator_indices']
+
+        # for mev rewards if we filter by address is possible to miss the block production
+        # since the locaiton label is the builder address and not the user address. To avoid this
+        # we use the location label to get the validator index and then use it to add extra
```

Comment:
Your logic for doing this does not make much sense though.

If the location label is the builder address then you are not interested in that query, no? You should **not** sum up these events if they are not tracked. The amount of ETH that goes in the block production to the builder is not something I am getting. So it should not be part of this